src/Physics/Resonance/XSection/BostedChristyEMPXSec.h

```diff
@@ -0,0 +1,79 @@
+//____________________________________________________________________________
+/*!
+
+\class    genie::BostedChristyEMPXSec
+
+\brief    An empirical fit of inclusive inelastic electron-proton cross sections in the
+          kinematic range of four-momentum transfer 0 < Q2 < 8 GeV^2 and final-state invariant 
+          mass 1.1 < W <3.1 GeV constrained by the high-precision longitudinal and 
+          transverse separated cross section measurements from Jefferson Lab Hall C. 
+          
+
+
+\author   Igor Kakorin <kakorin@jinr.ru>
+          Joint Institute for Nuclear Research
+
+\ref      M.E. Christy, P.E.Bosted, "Empirical fit to precision inclusive 
+          electron-proton cross sections in the resonance region", PRC 81 (2010) 055213
+
+\created  April 3, 2021
+
+\cpright  Copyright (c) 2003-2021, The GENIE Collaboration
+          For the full text of the license visit http://copyright.genie-mc.org
+          or see $GENIE/LICENSE
+*/
+//____________________________________________________________________________
+#ifndef _BOOSTED_CHRISTY_EM_PXSEC_H_
+#define _BOOSTED_CHRISTY_EM_PXSEC_H_
+
+#include "Framework/EventGen/XSecAlgorithmI.h"
+
+namespace genie {
+
+class BostedChristyEMPXSec : public XSecAlgorithmI {
+
+public:
+  BostedChristyEMPXSec();
+  BostedChristyEMPXSec(string config);
+  virtual ~BostedChristyEMPXSec();
+
+  // implement the XSecAlgorithmI interface
+  double XSec         (const Interaction * i, KinePhaseSpace_t k) const;
+  double Integral     (const Interaction * i) const;
+  bool   ValidProcess (const Interaction * i) const;
+
+  // overload the Algorithm::Configure() methods to load private data
+  // members from configuration options
+  void Configure(const Registry & config);
+  void Configure(string config);
+
+private:
+
+  void LoadConfig (void);
+  double sigmaR(int, double, double) const;
+  double sigmaNR(int, double, double) const;
+
+  double fbr[7][3];            ///< branching ratios of resonances
```

Comment:
The Genie naming convention is 
'f' followed by capital letter. So, change it to fBR.
Since now we are using C++14, make them std::array<double>.
Also, add a comment that describe the mapping from the entry to the resonance

Reply:
> 'f' followed by capital letter. So, change it to fBR.
> Since now we are using C++14, make them std::array.
Ok

> Also,` add a comment that describe the mapping from the entry to the resonance
Sorry, I don't understand what comment you mean.